src/Integration.Vsix/Suppression/LiveIssueFactory.cs

```diff
@@ -98,7 +97,6 @@ public LiveIssue Create(SyntaxTree syntaxTree, Diagnostic diagnostic)
             }
 
             var lineSpan = diagnostic.Location.GetLineSpan();
-            var relativeFilePath = FileUtilities.GetRelativePath(projectFilePath, lineSpan.Path);
```

Comment:
We no longer want a path relative to the csproj.

Reply:
It looks like this was the only place that used _GetRelativePath_ so it could be deleted.